login/service.go

```diff
@@ -633,97 +627,31 @@ func (keycloak *KeycloakOAuthProvider) getReferrerAndResponseMode(ctx context.Co
 
 // CreateOrUpdateIdentityInDB creates a user and a keycloak identity. If the user and identity already exist then update them.
 // Returns the user, identity and true if a new user and identity have been created
-func (keycloak *KeycloakOAuthProvider) CreateOrUpdateIdentityInDB(ctx context.Context, accessToken string, configuration Configuration) (*account.Identity, bool, error) {
+func (keycloak *KeycloakOAuthProvider) CreateOrUpdateIdentityInDB(ctx context.Context, accessToken string, idpProvider oauth.IdentityProvider, configuration Configuration) (*account.Identity, bool, error) {
```

Comment:
I can't see any creation or updation of Identity in auth DB. Maybe we should rename a function. 
I am not sure do we need to map RHD identity of a user in auth DB. Can you help me to understand @sbose78 ?

Reply:
True, we no longer create new records as part of this! Will address this.